hammer_cli_foreman_remote_execution.gemspec

```
@@ -14,9 +14,9 @@ Gem::Specification.new do |s|
   s.summary       = 'CLI for the Foreman remote execution plugin'
   s.description   = 'This gem provides CLI support for the Foreman remote execution plugin'
 
-  s.files =            `git ls-files`.split("\n")
-  s.test_files =       `git ls-files test`.split("\n")
-  s.extra_rdoc_files = `git ls-files doc`.split("\n") + Dir['README*', 'LICENSE']
+  s.files =            Dir['{lib,config,locale}/**/*', 'LICENSE', 'README.md']
+  s.test_files =       Dir["test/**/*"]
+  s.extra_rdoc_files = Dir['README*', 'LICENSE']
```

Should we just make it explicit, as done above?
```suggestion
  s.extra_rdoc_files = ['LICENSE', 'README.md']
```

---

Thanks, updated.